packages/build-utils/src/lambda.ts

```diff
@@ -47,6 +48,7 @@ interface GetLambdaOptionsFromFunctionOptions {
 
 export class Lambda {
   type: 'Lambda';
+  operationType?: string;
```

Comment:
Non-blocking: I got to thinking that if `operationType` is not required, then what is the default when the value is `undefined`? Maybe add a hint or something?

```suggestion
  operationType?: string; // default "API"
```